plugins/pa11y/readme.md

```diff
@@ -0,0 +1,30 @@
+# @dotcom-tool-kit/pa11y
+
+A plugin to run [Pa11y](https://github.com/pa11y/pa11y) accessibility tests. This plugin uses [Pa11y CI](https://github.com/pa11y/pa11y-ci) to run the tests.
+
+## Installation
+
+With Tool Kit [already set up](https://github.com/financial-times/dotcom-tool-kit#installing-and-using-tool-kit), install this plugin as a dev dependency:
+
+```sh
+npm install --save-dev @dotcom-tool-kit/pa11y
+```
+
+And add it to your repo's `.toolkitrc.yml`:
+
+```yaml
+plugins:
+    - '@dotcom-tool-kit/pa11y'
+```
+
+## Options
+
+| Key | Description | Default value |
+|-|-|-|
+| `configFile` | Path to the config file | `.pa11yci.js` |
+
+## Hooks
+
+| Event | Description |
+|-|-|
+| `test:review` | Runs `pa11y-ci` |
```

Comment:
**suggestion**: it might be helpful to also a column for Task, to show that this default hook is running `Pa11y` task

Reply:
**thought**: do we also want to include a description of where the hook will typically be called in a `Hook Description` column or should we leave that to the plugins which actually install the hooks?

Reply:
I am not sure what you mean here, please could you provide an example?

Reply:
We can use secret squirrel's docs as an [example](https://github.com/Financial-Times/dotcom-tool-kit/tree/main/plugins/secret-squirrel#tasks) - there is a column called Task which mentions that it would run the `SecretSquirrel` [Task](https://github.com/Financial-Times/dotcom-tool-kit/blob/main/plugins/secret-squirrel/src/tasks/secret-squirrel.ts#L7)